templates/ovndbcluster/bin/cleanup.sh

```diff
@@ -44,3 +44,11 @@ if [[ "$(hostname)" != "{{ .SERVICE_NAME }}-0" ]]; then
     # now that we left, the database file is no longer valid
     cleanup_db_file
 fi
+
+# Stop the DB server gracefully; this will also end the pod running script.
```

Comment:
(Side note, not necessarily the issue to resolve for this PR)

I wonder if now that we [have](https://github.com/openstack-k8s-operators/ovn-operator/pull/327/files) a proper trap handler set in the startup script, we should not move the `cleanup.sh` code there.